netbox/extras/models.py

```diff
@@ -88,7 +88,14 @@ class Webhook(models.Model):
     )
 
     class Meta:
-        unique_together = ('payload_url', 'type_create', 'type_update', 'type_delete',)
+        constraints = [
+            models.UniqueConstraint(
+                fields=[
+                    'payload_url', 'type_create', 'type_update', 'type_delete'
+                ],
+                name='webhook_webhooktype_payload_key'
```

Comment:
Here the constraint name would get a little long. Should I still adhere to the full naming scheme? That would render it as `extras_webhook_payload_url_type_create_type_update_type_delete_key` in its most verbose form, which I find supremely unreadable, but YMMV and I’m ready to change my mind.